javascript/javascript2/week1/median_average/m&a.js

```diff
@@ -0,0 +1,47 @@
+const housePrices = [3000000, 3500000, 1300000, 40000000, 100000000, 8000000, 2100000];
+function calculateAverage(value) {
+    let total = 0;
+    for (var i = 0; i < value.length; i++) {
+        total += value[i];
+    }
+    const average = (total / (value.length)).toFixed(2);
+    return average;
+
+}
+
+function calculateMedian(values) {
+    const arrSort = values.sort();
```

Comment:
by default sort won't work correctly with arrays of numbers, it only works with arrays of strings,

this documentation explains it best https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort

you have to call it with a function like this for it to work correctly
```
function compareNumbers(a, b) {
  return a - b;
}
```